chrome/webpack.config.js

```diff
@@ -10,21 +10,21 @@ const version = require("../package.json").version;
 module.exports = {
   entry: "./chrome/addon.js",
   output: {
-    filename: "./dist/chrome/background.js"
+    filename: "../dist/chrome/background.js"
```

Comment:
I wonder *what* changed between 3 and 4 that caused this to stop working. 🤷‍♂️ 🤷‍♀️

Reply:
It does seem sort of random! I tried a bunch of searches to try to figure out what was causing it, and whether it was webpack itself or the fact that webpack 4 needed the 4.5 upgrade of copy-webpack-plugin and *that* did something wonky ... but I didn't find any similar reports.

Reply:
Thanks for checking! Let's just blame ghosts. 👻